### ISSUE TITLE
redcarpet performs unnecessary escape for rendered tag

### ISSUE THREAD

**Issue by jylitalo** [vmg/redcarpet#414]
When I use rdiscount as markdown parser in jekyll, my liquid tag (at https://github.com/jylitalo/ylitalot-net/blob/master/_plugins/colorbox.rb) will properly render:
&lt;script>$(document).ready(function() {$('a[href$="_l.jpg"]').colorbox();});&lt;/script>

If I try to use redcarpet for the same task, it will come out as 
&lt;script>$(document).ready(function() {$(&amp;#39;a[href$=&amp;quot;_l.jpg&quot;]&amp;#39;).colorbox();});&lt;/script>
which on my case doesn't serve its purpose.

So the problem is that redcarpet somehow seems to escape text from my render method (' character into &amp;#39; and " to &amp;quot;), while rdiscount seems to trust that plugin produces proper html.

**Comment by mattberther** [comment]
This markdown

``` md
Testing

<script>
    var var1 = 'en';
    var var2 = '{{ site.url }}'
</script>

Testing
```

ends up producing the following HTML with Redcarpet 3.2.0

``` html

<p>Testing</p>

<p><script>
    var var1 = &#39;en&#39;;
    var var2 = &#39;&#39;
</script></p>

<p>Testing</p>
```

Not sure what Im missing -- how do I prevent Redcarpet from putting the <p> tags around the script tags?

**Comment by robin850** [comment]
@mattberther : Yep, this is a different bug, this has already been fixed with https://github.com/vmg/redcarpet/commit/0f7f50056c75e61ea6c9867b07b796a0f64ab564. Normally, this change is included in Redcarpet 3.2.2 (also see #440).
